src/submission/parseErrors/index.jsx

```diff
@@ -126,6 +127,30 @@ class ParseErrors extends Component {
   }
 }
 
+function errorResponseParser(errorResponse, uliNeeded) {
+  let apiErrorMessages =[]
+
+  errorResponse.forEach(function(errorsFound) {
+    if(errorsFound && errorsFound.errorMessages)
+    for (const [index, errorMessage] of errorsFound.errorMessages.entries()) {
+      let inputContent = errorMessage.inputValue
+
+      if (!errorMessage.inputValue) inputContent = <em>(blank)</em>
+ 
+      apiErrorMessages.push(
+        <tr key={index}>
+          <td>{errorsFound.rowNumber}</td>
+          {uliNeeded &&<td>{errorsFound.estimatedULI}</td>}
```

Comment:
Should evaluate to `null` to prevent rendering, rather than `false`:
```suggestion
          {uliNeeded ? <td>{errorsFound.estimatedULI}</td> : null}
```